make_pluralrules/tests/test.rs

```diff
@@ -26,3 +26,54 @@ fn simple_test() {
 
     assert_eq!(output_rs, output);
 }
+
+#[test]
+fn within_test() {
+    let text = String::from(
+"{
```

Comment:
Instead of escaping, please, use raw strings `r#""#` here: https://doc.rust-lang.org/rust-by-example/std/str.html